Formula/couchdb.rb

```diff
@@ -30,11 +30,18 @@ class Couchdb < Formula
     depends_on "help2man" => :build
   end
 
+  option "with-geocouch", "Build without GeoCouch spatial index extension"
+
   depends_on "spidermonkey"
   depends_on "icu4c"
   depends_on "erlang"
   depends_on "curl" if MacOS.version <= :leopard
 
+  resource "geocouch" do
+    url "https://codeload.github.com/couchbase/geocouch/tar.gz/couchdb1.3.x"
```

Comment:
Prefer GitHub archive rather than `codeload` URLs. We should probably add an audit for that. Also, do we not have a tagged release we can use?